relay-server/src/services/processor/span/processing.rs

```diff
@@ -47,6 +47,13 @@ pub fn process(
     addrs: &Addrs,
     buffer_guard: &BufferGuard,
 ) {
+    if !state
+        .project_state
+        .has_feature(Feature::ExtractSpansFromEvent)
+    {
+        return;
+    }
```

Comment:
@phacops this seems wrong:

1. do we disallow standalone span processing when the span _extraction_ flag is disabled?
2. If so, we should reject the span in `span::filter`.